supported_platforms.md

```diff
@@ -0,0 +1,55 @@
+# Microsoft Azure IoT SDKs for Java
+
+This SDK is tested nightly on Windows 10, Ubuntu 1604, and on an Android emulator. For additional details
+for each tested platform, see the respective sections below. 
+
+While this SDK contains several different packages, each package supports the same platforms and JDKs.
+
+## Supported JDK versions
+
+Currently, this SDK supports JDK versions 8 and 11. This SDK may work with other versions, but there are no guarantees 
+that they will always work for those versions nor are there guarantees that we will fix bugs that are only present on
+those versions.
+
+
+## Windows 10
+
+Nightly test platform details:
+Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
+Java version: 1.8.0_292, vendor: AdoptOpenJDK, runtime: C:\hostedtoolcache\windows\Java_Adopt_jdk\8.0.292-10\x64\jre
+Default locale: en_US, platform encoding: Cp1252
+OS name: "windows server 2016", version: "10.0", arch: "amd64", family: "windows"
+
+Note that, while we only directly test on Windows 10, we do support other Windows versions as well.
+
+## Ubuntu 1604
+
+Nightly test platform details:
+Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
+Java version: 1.8.0_292, vendor: AdoptOpenJDK, runtime: /usr/lib/jvm/adoptopenjdk-8-hotspot-amd64/jre
+Default locale: en_US, platform encoding: UTF-8
+OS name: "linux", version: "4.15.0-1113-azure", arch: "amd64", family: "unix"
+
+Note that, while we only directly test on Ubuntu 1604, we do generally support other popular linux distributions. 
+
+## Android
+
+This SDK supports Android API versions 24, 25, 26, 27, and 28.
+
+The android emulator image used for nightly tests is "system-images;android-28;google_apis;x86_64"
```

Comment:
wouldn't simply specifying that we test against api version 28 be enough, or does this also lend some additional information? It might be easier for customers if we specify the exact info that they require rather than leaving them to parse it. What do you think?

Reply:
There are a lot of Android emulators at API level 28, so this does provide a bit more context. The benefit here would be that users could try out the same emulator that we use. Maybe it is a bit too much extra information, though?

Reply:
I personally couldn't understand how to interpret this string - should I be looking for this exact string in Android Studio? What if I use a different IDE? How does this string translate to what I need to correctly identify the emulator that our nightly pipeline uses?
I think adding some additional context on what the different parts of this string signify would be helpful here.

Reply:
Should we capitalize Android?

Reply:
Good point. I can remove this string since adding context to it would probably fall outside of the scope of this document's purpose